clients/go/test/grpc-app/main.go

```diff
@@ -4,7 +4,7 @@
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-// https://www.apache.org/licenses/LICENSE-2.0
+//     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Why (curiosity)? The website itself self-links with https ?

Reply:
IANAL, but using `https` could technically invalidate the license: https://www.apache.org/licenses/LICENSE-2.0#apply

I'm not sure it would hold up in court, but that link above is the license body we need to apply.